locale/tr.json

```diff
@@ -6,9 +6,9 @@
     "restarter": {
         "start_timeout": "zamanında başlatılamadı",
         "crash_detected": "Çökme tespit edildi",
-        "hang_detected": "hang detected",
-        "partial_hang_warn": "Due to a partial hang, this server will restart in 1 minute. Please disconnect now.",
-        "partial_hang_warn_discord": "Due to a partial hang, **%{servername}** will restart in 1 minute.",
+        "hang_detected": "Kısmi bir takılma algılandı",
```

Comment:
Does "Kısmi" mean Partial? If that's the case then its better to remove the work, not to confuse with the partial hang warnings.

Reply:
"hang" has a lot of means from Turkish. In This case "Takılma" is good. I deleted "Kısmi" side.
http://prntscr.com/12q28zd